dask/dataframe/tests/test_utils_dataframe.py

```diff
@@ -229,6 +229,16 @@ def test_meta_nonempty_index():
         assert idx1.name == idx2.name
     assert res.names == idx.names
 
+    levels = [pd.Int64Index([1], name='a'),
+              pd.CategoricalIndex(data=['b'], categories=['b'], name='b')]
```

Comment:
I think this has to be `data=[], categories`[]` for it to fail without your fix.

Reply:
Without my fix this test fails with `ValueError: Level values must be unique: ['b', 'b'] on level 1` since the data is duplicated in line https://github.com/fjetter/dask/blob/0421127c6942f4dd904b2c2badfa5fcf3302a5b7/dask/dataframe/utils.py#L338

Reply:
Sorry, you're correct.